ci/build.py

```diff
@@ -346,15 +429,21 @@ def use_cache():
             tag = get_docker_tag(platform=platform, registry=args.docker_registry)
             if use_cache():
                 load_docker_cache(tag=tag, docker_registry=args.docker_registry)
-            build_docker(platform, docker_binary, args.docker_registry, num_retries=num_docker_build_retires)
+            build_docker(platform, docker_binary, args.docker_registry, num_retries=args.docker_build_retries,
+                         use_cache=use_cache())
             if args.build_only:
                 continue
-            build_platform = "build_{}".format(platform)
-            cmd = ["/work/mxnet/ci/docker/runtime_functions.sh", build_platform]
+            git_cleanup()
```

Comment:
This is good, but maybe we should issue a warning here with confirmation.

Reply:
Actually, how this would work when all platforms are build one after another? Only the latest artifacts survive?

Reply:
How would this work if for example I want to test a new file locally that hasn't yet been added to git?